src/test/java/org/jsmart/zerocode/core/tests/H2DBTest.java

```diff
@@ -0,0 +1,56 @@
+package org.jsmart.zerocode.core.tests;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.CoreMatchers.*;
+
+import java.sql.*;
+
+public class H2DBTest {
+
+    @Before
+    public void setUp() throws Exception{
+
+        Connection connection = createAndGetConnection();
+        Statement statement = connection.createStatement();
+
+        String createTableSql=  "CREATE TABLE IF NOT EXISTS CUSTOMER (id number, name varchar(20))";
+        statement.executeUpdate(createTableSql);
+
+        String insertSql= "INSERT into CUSTOMER values (1, 'Jeff Bejo')";
+        statement.executeUpdate(insertSql);
+
+        statement.close();
+        connection.close();
+    }
+
+    @Test
+    public void testH2DB() throws Exception{
+
+        Connection connection = createAndGetConnection();
+        Statement statement = connection.createStatement();
+        String selectSql=  "Select * from CUSTOMER";
+        ResultSet resultSet= statement.executeQuery(selectSql);
+        assertThat(resultSet, is(notNullValue()));
+
+        int count = -1;
+        if (resultSet.next())
+        {
+            count = resultSet.getInt(1);
+        }
+
+        assertThat(count,is(1));
```

Comment:
nice one. +1

Reply:
@dinesh76in , Is travis CI build is green with latest master merged to this branch?

Reply:
I checked the build locally ....but not the travis CI , do you have the url ?